Config/droplet.json

```diff
@@ -3,19 +3,5 @@
     "client": "engine",
     "console": "terminal",
     "log": "console",
-    "hash": "crypto",
-    "cipher": "crypto",
-    "middleware": {
```

Comment:
middleware should remain but with no differentiation b/t client and server. just one server array.